doc/reference/usermode/syscalls.rst

```diff
@@ -241,7 +241,7 @@ passed in.  This includes:
 * Any other arguments that have a limited range of valid values.
 
 Verification functions involve a great deal of boilerplate code which has been
-made simpler by some macros in ``kernel/include/syscall_handlers.h``.
+made simpler by some macros in ``kernel/include/syscall_handler.h``.
```

Comment:
I don't recall when this got moved but it's in `include/syscall_handler.h`, it's no longer in the private kernel include directory.

Reply:
Good catch. Fixed and replaced with `:zephyr_file:` so the doc build will fail if this goes stale again.